### ISSUE TITLE
SVG doesn't handle mspace with background color properly

### ISSUE THREAD

**Issue by dpvc** [mathjax/MathJax#991]
In SVG output, 

```
<math xmlns="http://www.w3.org/1998/Math/MathML" background="yellow">
  <mpadded>
    <mspace mathbackground="red" width="50px" height="30px" depth="30px" />
  </mpadded>
  <mpadded>
    <mspace mathbackground="green" width="10px" height="50px" depth="10px" />
  </mpadded>
</math>
```
the bounding box for the equation is not correct.  Switching the `<mspace>` elements for `<mpadded>` ones fixes the problem.  (Switching one at a time gives other incorrect results.)

**Comment by dpvc** [comment]
This turns out to be due to the fix for #800, and the problem is resolved in the `issue991` branch of my fork of MathJax.

**Comment by dpvc** [comment]
==> Merged
